shared/src/main/scala/amf/core/internal/errorhandling/WarningOnlyHandler.scala

```diff
@@ -7,6 +7,7 @@ import org.yaml.model.{SyamlException, YError}
 
 case class WarningOnlyHandler(parent: AMFErrorHandler) extends RuntimeWrapperErrorHandler(parent: AMFErrorHandler) {
 
+  /*
```

Comment:
Delete, don't comment

Reply:
Thanks, the comment make me notice that the report method were not being override to change severity to warning